### DISCUSSION TITLE
Question about Van Genuchten and Maulem

### DISCUSSION THREAD

**Discussion by zorroxt1118** [Anura3D/Anura3D_OpenSource#201] [Q&A]
I want to use Van Genuchten and Maulem to define the SWRC and HCC for the unsaturated soil. When selected these two equations in the GiD software. The two equations are both defined by Sr, Ss, reference pressure and lambda. My question is should I input identical parameters for both Van Genuchten and Maulem? I looked up the scientific manual, the Maulem equation has variables of S_L from VG and lambda. Then the two equations should be related. Why they need to be input twice?

**Comment by LongTIme1111** [comment]
Hi  
what is the scientific manual you mentioned, I also wang to use this part, but I don't know what's mean of tha parameter of lambda and refernce pressure, are those mentioned in the scientific manual?

**Comment by zorroxt1118** [comment]
Hi, here I attached the link.
https://github.com/Anura3D/Anura3D_OpenSource/blob/main/manuals/ScientificManual_2022.pdf
